layouts/partials/button/_button.scss

```diff
@@ -71,7 +71,10 @@ $button-hamburger-hover-br: transparent;
   color: $button-primary-hover-c;
 }
 
-.button-primary > svg {
+.button-primary > svg,
```

Comment:
With a `.button` base class you just only need to do it like this, so all buttons will get the same properties for svgs:

```
.button > svg {
  ...
}
```